tests/org.jboss.tools.jst.ui.bot.test/pom.xml

```diff
@@ -19,6 +18,9 @@
 				<configuration>
 					<testSuite>org.jboss.tools.jst.ui.bot.test</testSuite>
 					<testClass>org.jboss.tools.jst.ui.bot.test.AllJSTTests</testClass>
+					<systemProperties>
+						<org.osgi.framework.bundle.parent>ext</org.osgi.framework.bundle.parent>
+					</systemProperties>
 					<dependencies combine.children="append">
```

Comment:
you can remove all depedencies here. Doesnt matter if you do mvn verify or specify installBase folder, you will always get full Devstudio